Docker/entrypoint.sh

```diff
@@ -23,4 +20,50 @@ if [ -n "$CRON_MIN" ]; then
 	crontab -l | sed -r "\\#FreshRSS#s#^[^ ]+ #$CRON_MIN #" | crontab -
 fi
 
+if [ -n "$FRESHRSS_INSTALL" ]; then
+	php -f ./cli/do-install.php -- \
+		$(echo "$FRESHRSS_INSTALL" | sed -r 's/[\r\n]+/\n/g' | paste -s -) \
+		1>/tmp/out.txt 2>/tmp/err.txt
+	EXITCODE=$?
+	cat /tmp/out.txt | grep -v 'Remember to'
+	cat /tmp/err.txt | grep -v 'Please use' 1>&2
```

Comment:
```suggestion
	< /tmp/out.txt grep -v 'Remember to'
	< /tmp/err.txt grep -v 'Please use' 1>&2
```

Reply:
I was not familiar with this syntax

Reply:
For the record: https://github.com/koalaman/shellcheck/wiki/SC2002
I was familiar with a syntax like `wc < tmp.txt` but not the other way around like `< tmp.txt wc`
I have to admit that can better like the cat version for its consistency. For instance, we still need one further below.
I did not know UUOC was a thing, so I have learned something !
Changed; good now?